AISKU/scripts/publishReleaseToCdn.ps1

```diff
@@ -570,9 +598,6 @@ if ([string]::IsNullOrWhiteSpace($jsSdkDir) -eq $true) {
 $fileTimeStamp = ((get-date).ToUniversalTime()).ToString("yyyyMMddThhmmss")
 $logFile = "$logDir\publishReleaseCdnLog_$fileTimeStamp.txt"
 
-$cacheControl1Year = "public, max-age=31536000, immutable";
```

Comment:
is cache time not needed anymore?

Reply:
I moved it to the top of the file - it's on line 16 now 😃